Source/WebCore/platform/network/cocoa/ResourceResponseCocoa.mm

```diff
@@ -174,7 +174,7 @@ static inline AtomString extractHTTPStatusText(CFHTTPMessageRef messageRef)
         }
         if (messageRef && initLevel == AllFields) {
             m_httpStatusText = extractHTTPStatusText(messageRef);
-            m_httpVersion = String(adoptCF(CFHTTPMessageCopyVersion(messageRef)).get()).convertToASCIIUppercase();
+            m_httpVersion = AtomString(adoptCF(CFHTTPMessageCopyVersion(messageRef)).get()).convertToASCIIUppercase();
```

Comment:
A real shame to not convert to uppercase as part of allocating the first AtomString, so we might have a temporary one that needs to be deleted.